examples/example_mountain_car.py

```diff
@@ -106,12 +102,14 @@ def objective(ind, seed, n_total_steps):
             warnings.filterwarnings(
                 "ignore", message="invalid value encountered in double_scalars"
             )
-            cum_reward_all_episodes = inner_objective(f, seed, n_total_steps, render=False)
+            cum_reward_all_episodes = inner_objective(
+                f, seed, n_trials_per_individual, n_total_steps, render=False
+            )
 
         # more episodes are better, more reward is better
         n_episodes = float(len(cum_reward_all_episodes))
         mean_cum_reward = np.mean(cum_reward_all_episodes)
-        ind.fitness = n_episodes + 1.0 * mean_cum_reward
+        ind.fitness = n_episodes / n_trials_per_individual + mean_cum_reward
```

Comment:
The first summand is a bit confusing, but renaming the variable as suggested above is enough I guess